src/onnxflow/justbuildit/ignition.py

```diff
@@ -478,7 +521,7 @@ def validate_inputs(inputs: Dict):
 
 def identify_model_type(model) -> build.ModelType:
     # Validate that the model's type is supported by build_model()
-    # and assign a ModelType tag
+    # and assign a ModelTye tag
```

Comment:
nit: typo

Reply:
Weird! Fixed in https://github.com/groq/mlagility/pull/306/commits/ee95b50188bfc85d8aa53180868b5ede05f3878b